mod/poco.php

```diff
@@ -121,10 +122,12 @@ function poco_init(App $a) {
 			if (isset($contact['account-type'])) {
 				$contact['contact-type'] = $contact['account-type'];
 			}
-			$about = DI::cache()->get("about:" . $contact['updated'] . ":" . $contact['nurl']);
+
+			$cacheKey = 'about:' . $contact['nick'] . ':' . DateTimeFormat::utc($contact['updated'], DateTimeFormat::ATOM);
```

Comment:
The `nick` field is not unique. Of course it is unlikely that the `updated` date is the same for two contacts with the same `nick`, but nonetheless it is better to take some different value.

Reply:
The nick is unique among the local users which this cache key is about. About, get it? 😜

Reply:
Looking at that code fragment, I guess we can fully omit the caching here.

Reply:
It's because of the `BBCode::convert` that's potentially expensive. Why remove caching from a public endpoint that can possibly be abused?